### PR TITLE
Update Dependecies to lasted versions

### PR DESCRIPTION
**Pull request by nimaa77**
TODO:
- [x] root package.json
- [x] razzle package
- [x] babel-preset-razzle
- [x] razzle-plugin-php
- [x] create-razzle-app
- [x] razzle-plugin-eslint

### Breaking Changes
nothing, I did my best to update packages to the versions that support node version 8

updating these plugins will create breaking changes, so I'm going to open another PR for these
razzle-plugin-mdx
razzle-plugin-scss
razzle-plugin-typescript

### CODE REVIEW

**Comment by fivethreeo** [comment]
Should we add some github actions to do matrix tests and simplify releases?

**Comment by fivethreeo** [comment]
Oh, there is github actions. But not for releases.

**Comment by nimaa77** [comment]
yeah that's cool we have the same thing for after.js but for that, we need @jaredpalmer to add npm token to GitHub so we can have a pipeline for releases and other stuff ...

**Comment by fivethreeo** [comment]
Also webpack 5 minimum node version is 10.x, since it uses the spread operator and is not transpiled. So maybe drop support below 10.x next major razzle razzle release.

**Comment by nimaa77** [comment]
yeah, that's what we are going to do for the next major version.

did you send the email to Jared palmer? you can find his email from his GitHub account page

**Review by fivethreeo — COMMENTED** [review]
Looks good to me

**Comment by fivethreeo** [comment]
Email sent

File: package.json [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
-    "jest": "^24.9.0",
-    "lerna": "2.10.1",
-    "lerna-changelog": "^0.7.0",
-    "lint-staged": "^4.0.0",
-    "prettier": "^1.4.4",
-    "ps-tree": "1.1.0",
-    "semver": "5.5.1",
-    "shelljs": "^0.8.2"
+    "eslint": "^6.8.0",
+    "husky": "^4.2.3",
```
Comment (kkalavantavanich) [inline package.json:7]: Make sure to update breaking `husky` changes. You can see the example for 2.7 here: https://github.com/jaredpalmer/razzle/pull/1059

**Comment by wickedev** [comment]
In order to make github action success on Windows, the `test/jest.config.json` file must be changed as shown below. https://github.com/facebook/jest/issues/7914#issuecomment-464352069

```json
{
  "roots": ["<rootDir>/tests"],
  "collectCoverageFrom": ["**/*.js"],
  "testMatch": [
    "<rootDir>/tests/**/*(*.)@(spec|test).[tj]s?(x)"
  ]
}
```

**Comment by wickedev** [comment]
I made a pull request to fix the `next` branch's github action. https://github.com/jaredpalmer/razzle/pull/1217

File: package.json (hunk 1 above)
Comment (nimaa77) [reply package.json:7]: ❤️

**Pull request merged**